integration-test/src/main/scala/com/cloudera/livy/test/framework/BaseIntegrationTestSuite.scala

```diff
@@ -97,17 +97,18 @@ abstract class BaseIntegrationTestSuite extends FunSuite with Matchers {
     }
   }
 
-  test("initialize test cluster") {
+  override def beforeAll() {
```

Comment:
> really hard to debug it, since the exception happens during discovery and scalatest doesn't generated good logs in that case.

I injected an exception in beforeAll and it's giving me the full stacktrace. What is your expectation of good logs? Do you mean info()/note() are not showing up for beforeAll()?
`*** RUN ABORTED *** (14 seconds, 853 milliseconds)
  java.lang.Exception: throw form runLivy()
  at com.cloudera.livy.test.framework.MiniCluster.runLivy(MiniCluster.scala:254)
  at com.cloudera.livy.test.framework.MiniCluster.deploy(MiniCluster.scala:234)
  at com.cloudera.livy.test.framework.Cluster$.liftedTree1$1(Cluster.scala:102)
  at com.cloudera.livy.test.framework.Cluster$.cluster$lzycompute(Cluster.scala:90)
  at com.cloudera.livy.test.framework.Cluster$.cluster(Cluster.scala:88)
  at com.cloudera.livy.test.framework.Cluster$.get(Cluster.scala:116)
  at com.cloudera.livy.test.framework.BaseIntegrationTestSuite.beforeAll(BaseIntegrationTestSuite.scala:103)
  at org.scalatest.BeforeAndAfterAll$class.beforeAll(BeforeAndAfterAll.scala:187)
  at com.cloudera.livy.test.framework.BaseIntegrationTestSuite.beforeAll(BaseIntegrationTestSuite.scala:49)
  at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:253)
`